### PR TITLE
Add and run tflint on tf examples

### PR DESCRIPTION
**Pull request by NissesSenap**
* Add.tflint.hcl
* Run lint on
  * github
  * install
  * multi-env
* Update .gitignore to ignore .terraform.lock.hcl
* run terraform fmt
* Add lint to make
* Add make lint to github action e2e

### CODE REVIEW

File: Makefile
Comment (phillebaba) [inline Makefile:45]: I get that we need to remove the data sources as they will never pass the lint check. Can you capitalize the variable names?

**Comment by phillebaba** [comment]
@NissesSenap see my comment, could you also update the branch?

File: Makefile
Comment (NissesSenap) [reply Makefile]: Done, also changed the name on the variables and made the the actual script working :)

**Comment by NissesSenap** [comment]
@phillebaba fixed

**Comment by phillebaba** [comment]
I am going to hold off with merging this as there are two other PRs that will be blocked if this is merged before. As soon as they are merged I will merge this.

**Review by phillebaba — APPROVED** [review]
LGTM :rose:

**Pull request merged**
